hazelcast/src/main/java/com/hazelcast/internal/diagnostics/Diagnostics.java

```diff
@@ -207,7 +219,12 @@ public void start() {
             return;
         }
 
-        this.diagnosticsLogFile = new DiagnosticsLogFile(this);
+        if (stdout) {
```

Comment:
Can you convert this into using a ternary operator?

```
this.diagnosticsLogs = stdOut ? new DiagnosticsStdout(this) :  new DiagnosticsLogFile(this);
```
It makes the code less verbose.